packages/gdl-frontend/components/WelcomeTutorial/WelcomeTutorial.js

```diff
@@ -0,0 +1,379 @@
+        activeStep: this.state.activeStep + 1,
+        direction: 'left'
+      });
+    }
+  }
+
+  handleMove(touchMoveEvent: touchMoveEventType, mobile: boolean) {
+    const clientX = touchMoveEvent.targetTouches[0].clientX;
+    if (this.state.beingTouched && mobile) {
+      const currTime = Date.now();
+      const elapsed = currTime - this.state.timeOfLastDragEvent;
+      const velocity = (20 * (clientX - this.state.prevTouchX)) / elapsed;
+      let deltaX = clientX - this.state.touchStartX + this.state.originalOffset;
+      if (deltaX < -200) {
+        this.handleNext();
+        deltaX = 0;
+        this.setState({ beingTouched: false });
+      } else if (deltaX > 200) {
+        if (this.state.activeStep > 0) this.handleBack();
+        deltaX = 0;
+        this.setState({ beingTouched: false });
+      }
+      this.setState({
+        left: deltaX,
+        velocity,
+        timeOfLastDragEvent: currTime,
+        prevTouchX: clientX
+      });
+    }
+  }
+  handleTouchEnd(mobile: boolean) {
+    if (mobile) {
+      this.setState({
+        velocity: this.state.velocity,
+        touchStartX: 0,
+        beingTouched: false,
+        intervalId: window.setInterval(this.animateSlidingToZero.bind(this), 33)
+      });
+    }
+  }
+
+  componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions.bind(this));
+  }
+
+  updateWindowDimensions() {
+    this.setState({ width: window.innerWidth, height: window.innerHeight });
+  }
+
+  handleClickOpen() {
+    this.setState({
+      open: true,
+      activeStep: 0
+    });
+  }
+
+  handleBack() {
+    if (this.state.activeStep === 1) {
+      this.setState({
+        isHidden1: 'flex',
+        isHidden2: 'none',
+        isHidden3: 'none',
+        direction: 'right',
+        activeStep: this.state.activeStep - 1
+      });
+    }
+    if (this.state.activeStep === 2) {
+      this.setState({
+        isHidden1: 'none',
+        isHidden2: 'flex',
+        isHidden3: 'none',
+        direction: 'right',
+        activeStep: this.state.activeStep - 1
+      });
+    }
+  }
+
+  render() {
+    let mobile: boolean = this.state.width < fullscreenWidth;
+    return (
+      <Dialog
+        fullScreen={mobile}
+        open={this.state.open}
+        onClose={this.handleClose.bind(this)}
+      >
+        <div
+          style={Styles.dialogWindow}
+          onTouchStart={touchStartEvent =>
+            this.handleTouchStart(touchStartEvent, mobile)
+          }
+          onTouchMove={touchMoveEvent =>
+            this.handleMove(touchMoveEvent, mobile)
+          }
+          onTouchEnd={() => {
+            this.handleTouchEnd(mobile);
+          }}
+        >
+          <Slide
+            style={{ height: '100%' }}
+            direction={this.state.direction}
+            in={this.state.activeStep === 0}
+          >
+            <div
+              style={{
+                display: this.state.isHidden1,
```

Comment:
Her kan du faktisk sammenligne på `this.state.activeStep` med f.eks `display: this.state.activeStep === 0 ? 'flex' : 'none'`. Da kan du potensielt fjerne hele 3 states `state.isHidden1, 2, 3`!